tests/dem/particle_particle_full_contact.cc

```diff
@@ -0,0 +1,191 @@
+              << output.force[i][2] << " torque is " << std::setw(10)
+              << output.torque[i][0] << " " << std::setw(10)
+              << output.torque[i][1] << " " << std::setw(10)
+              << output.torque[i][2] << " overlap is " << std::setw(10)
+              << output.overlap[i] << std::endl;
+    }
+}
+
+
+template <int dim, typename PropertiesIndex>
+void
+test()
+{
+  // Creating the mesh and refinement
+  parallel::distributed::Triangulation<dim> triangulation(MPI_COMM_WORLD);
+  int                                       hyper_cube_length = 1;
+  GridGenerator::hyper_cube(triangulation,
+                            -1 * hyper_cube_length,
+                            hyper_cube_length,
+                            true);
+  int refinement_number = 2;
+  triangulation.refine_global(refinement_number);
+  MappingQ<dim> mapping(1);
+
+
+  // Defining general simulation parameters
+  DEMSolverParameters<dim>                              dem_parameters;
+  Parameters::Lagrangian::LagrangianPhysicalProperties &lagrangian_prop =
+    dem_parameters.lagrangian_physical_properties;
+  Parameters::Lagrangian::ModelParameters &model_param =
+    dem_parameters.model_parameters;
+
+  Tensor<1, dim> g{{0, 0, 0}};
+  double         dt                                               = 0.00001;
+  double         particle_diameter                                = 0.005;
+  unsigned int   output_step                                      = 10;
+  lagrangian_prop.particle_type_number                            = 1;
+  lagrangian_prop.youngs_modulus_particle[0]                      = 50000000;
+  lagrangian_prop.poisson_ratio_particle[0]                       = 0.3;
+  lagrangian_prop.restitution_coefficient_particle[0]             = 0.5;
+  lagrangian_prop.friction_coefficient_particle[0]                = 0.5;
+  lagrangian_prop.rolling_viscous_damping_coefficient_particle[0] = 0.5;
+  lagrangian_prop.rolling_friction_coefficient_particle[0]        = 0.1;
+  lagrangian_prop.surface_energy_particle[0]                      = 0.;
+  lagrangian_prop.hamaker_constant_particle[0]                    = 0.;
+  lagrangian_prop.density_particle[0]                             = 2500;
+  model_param.rolling_resistance_method =
+    RollingResistanceMethod::constant_resistance;
+
+  const double neighborhood_threshold = std::pow(1.3 * particle_diameter, 2);
+
+  // Defining particle handler
+  Particles::ParticleHandler<dim> particle_handler(
+    triangulation, mapping, DEM::get_number_properties<PropertiesIndex>());
+
+  // Creating containers manager
+  DEMContactManager<dim, PropertiesIndex> contact_manager;
+
+  // Finding cell neighbors list
+  typename dem_data_structures<dim>::periodic_boundaries_cells_info
+    dummy_pbc_info;
+  contact_manager.execute_cell_neighbors_search(triangulation, dummy_pbc_info);
+
+  // Setting initial properties of particle 0 and 1
+  Point<dim>                       position0 = {0.4, 0, 0};
```

Comment:
```suggestion
  Point<dim>                       position_0 = {0.4, 0, 0};
```

do the same for the following variables